storage/prom-pushgateway/pushgateway.go

```diff
@@ -47,12 +47,13 @@ func (c *promClient) Dump(tags map[string]string, benchmarks ...performance.Benc
 	log.Debugf("getting metrics")
 	metrics := getMetrics(labels)
 	for _, b := range benchmarks {
-		tmpValues := append([]string{b.Name}, values...)
+		bench := b.Benchmark
+		tmpValues := append([]string{bench.Name}, values...)
 
 		log.Debugf("observing for the benchmark: %+v", b)
-		metrics[storage.PerOpSeconds].WithLabelValues(tmpValues...).Observe(float64(time.Duration(b.NsPerOp).Seconds()))
-		metrics[storage.PerOpAllocBytes].WithLabelValues(tmpValues...).Observe(float64(b.AllocedBytesPerOp))
-		metrics[storage.PerOpAllocs].WithLabelValues(tmpValues...).Observe(float64(b.AllocsPerOp))
+		metrics[storage.PerOpSeconds].WithLabelValues(tmpValues...).Observe(float64(time.Duration(bench.NsPerOp).Seconds()))
```

Comment:
The type conversion to `float64` isn't needed, the return value from [Seconds](https://godoc.org/time#Duration.Seconds) is already that type.

Reply:
Done